.github/workflows/automerge.yml

```diff
@@ -0,0 +1,35 @@
+name: Dependabot Automerge
```

Comment:
I'm curious. Why create a new workflow for automerging? Why not add a new job to the ci workflow so that it won't automerge if a different job on the ci workflow fails?

Reply:
I'm good with adding it to the CI workflow. The only reason I created a new workflow was so that it would only be triggered on dev branch merges.

Reply:
Ah, good point. I only worry about automerging if some checks fail in the ci. does the GHA only execute if all jobs in the workflow succeed or all workflows that ran on the commit succeed?

Reply:
I only have it running the tests that are run with `npm test`. Are there more checks that should be run before we merge dependabot PRs?

Reply:
Yes there are other checks, like the terraform plan check, the docker build check, the lint check, etc. I would maybe put the GHA for automerging inside the ci just so that you don't have to rerun all these jobs. Any other thoughts?

Also do we want to automerge any PR that passes checks? Should we only automerge dependabot update PRs?

Reply:
Also, it should work fine in the CI workflow, since it will only merge dependabot PRs with the given criteria. And from what I understand, dependabot is configured to merge to the dev branch. The only concern I would have is if for some reason dependabot creates a PR to merge to master or stg, and then it would merge to master or stg without us wanting to.

Reply:
You can put if statements on a job to make sure it'll only run on PRs to `dev` and maybe also have it dependent on all other jobs